test/test/jfr/JfrTests.java

```diff
@@ -195,4 +197,9 @@ private boolean containsSamplesOutsideWindow(TestProcess p) throws Exception {
             return entryEnd.isBefore(event.getStartTime());
         });
     }
+
+    private static boolean checkJdkVersionEarlierThan11() {
+        String javaVersion = System.getProperty("java.version");
+        return javaVersion.startsWith("1.") || Integer.parseInt(javaVersion.split("\\.")[0]) < 11;
+    }
```

Comment:
This returns the version of the host JDK (test runner), which may differ from the JDK version used by the test itself.
You may find the version from the JFR recording.

Reply:
790b08607ecee5f13fdc4a5bc63a2b3516208f5b